src/components/Menu/Menu.style.ts

```diff
@@ -83,30 +31,30 @@ export type MenuOptions = {
 };
 
 export const optionsStyle = ({ menuPosition }: MenuOptions) => (theme: Theme) => css`
-    max-height: 400px;
-    overflow-y: scroll;
-    position: absolute;
-    top: ${rem(48)};
-    left: ${menuPosition === 'left' ? 0 : 'initial'};
-    right 0;
-    width: ${rem(148)};
-    height: auto;
-    background-color: ${theme.palette.white};
-    box-shadow: 0px 0px ${rem(16)} grey;
-    display: flex;
-    flex-direction: column;
-    justify-content: space-evenly;
-    text-align: center;
-    border-radius: ${rem(4)};
-    z-index: 1;
-    & > button {
-      padding: ${rem(8)} 0;
-      height: ${rem(48)};
-      margin-left: 0;
-      font-size: ${theme.typography.fontSizes['14']};
-    }
+  max-height: 400px;
+  overflow-y: scroll;
+  position: absolute;
+  top: ${rem(48)};
+  left: ${menuPosition === 'left' ? 0 : 'initial'};
+  right: 0;
+  width: ${rem(148)};
+  height: auto;
+  background-color: ${theme.palette.white};
+  box-shadow: ${theme.elevation['02']};
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  text-align: center;
+  border-radius: ${rem(4)};
+  z-index: 1;
+  & > button {
+    padding: ${rem(8)} 0;
+    height: ${rem(48)};
```

Comment:
the buttons are stretched when lots of options are added.
![image](https://user-images.githubusercontent.com/36039128/102506040-61c71180-408b-11eb-8f81-f16889c5ee5f.png)

adding `min-height:  ${rem(48)};` seems to solve the problem

Reply:
Another problem that arises by adding lots of options is that lots of them exceeds the parent container and are not visible 
![image](https://user-images.githubusercontent.com/36039128/102507238-b1f2a380-408c-11eb-80fb-f34c0a66225a.png) 
![image](https://user-images.githubusercontent.com/36039128/102507391-dc446100-408c-11eb-9309-b0c7b7142f6e.png)

Reply:
i think i fixed what you are mentioning here but please recheck the latest version

Reply:
Works fine! thank you